scripts/deploy.py

```diff
@@ -1,7 +1,28 @@
 #!/usr/bin/python3
```

Comment:
should this file be named specific to subscription manager eg. `deploy_subscription_manager.py` OR do we envision this single script being more customizable once other contracts are deployed from this repos.

Reply:
You're totally right.

Reply:
✅